.github/dependabot.yml

```diff
@@ -0,0 +1,27 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+# GitHub Actions
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+      interval: "weekly"
+  commit-message:
+      prefix: ":seedling:"
+  labels:
+    - "ok-to-test"
+# Go
+- package-ecosystem: "gomod"
+  directory: "/"
+  schedule:
+    interval: "weekly"
+  ignore:
+  # Ignore k8s modules as they are upgraded manually
```

Comment:
We have some more excludes in core CAPI: https://github.com/kubernetes-sigs/cluster-api/blob/main/.github/dependabot.yml

Just in case you want to exclude them as well

Reply:
Ah sounds good